rmf_demo_plugins/rmf_ignition_plugins/src/TeleportDispenser.cpp

```diff
@@ -153,6 +160,13 @@ void TeleportDispenserPlugin::place_on_entity(EntityComponentManager& ecm,
       components::WorldPoseCmd(ignition::math::Pose3<double>()));
   }
   ecm.Component<components::WorldPoseCmd>(to_move)->Data() = new_pose;
+
+  // For Ignition slotcar plugin to know when an item has been dispensed to it
+  // Necessary for TPE Plugin
+  ignition::msgs::UInt64_V dispense_msg;
+  dispense_msg.add_data(google::protobuf::uint64(base)); //Conversion from Entity -> uint64
+  dispense_msg.add_data(google::protobuf::uint64(to_move));
```

Comment:
I wish there was a more explicative message type but I guess it would be overkill to compile our own message for such a narrow use case